docs/Reference/Config-Items.md

```diff
@@ -16,6 +20,26 @@ Network configuration items are specified in the Hyperledger Besu genesis file.
 | `evmStackSize`      | Maximum stack size. Specify to increase the maximum stack size in private networks with very complex smart contracts. Default is `1024`. |
 
 
+## Genesis Block Parameters  
```

Comment:
Is `Block Parameters` the real name or is this something that we came up with?

Also, I think it would be useful to add an example of the meaning of these parameters in one of the consensus algorithms used (are we explaining them somewhere else?). E.g:
```
When using PoW, the difficulty represents how hard it is for a miner to create a block. The lower the value, the faster the node will generate blocks.
```

Reply:
The protocol names are linked and explanations are included there

Reply:
Block Parameters used to differentiate those items from the config items